src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -782,8 +777,20 @@ export class ServerView extends DirMixin(PolymerElement) {
     this.dispatchEvent(makePublicEvent('RemoveAccessKeyRequested', {accessKeyId: accessKey.id}));
   }
 
-  setServerTransferredData(totalBytes) {
-    this.totalInboundBytes = totalBytes;
+  _getInboundBytesUnit(totalBytes, language) {
```

Comment:
Perhaps `_formatInboundBytesUnit` and `_formatInboundBytesValue` is clearer.

The unit is bytes, so "get unit" isn't quite accurate.

Reply:
Done in 435c5a5a56dab1230b3f11762438144ba3f5d7cf